core/metadata.jvm/test/org/jetbrains/kotlin/metadata/jvm/JarContentTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.metadata.jvm
+
+import jdk.internal.org.objectweb.asm.ClassReader
+import jdk.internal.org.objectweb.asm.ClassReader.SKIP_FRAMES
+import jdk.internal.org.objectweb.asm.ClassVisitor
+import jdk.internal.org.objectweb.asm.MethodVisitor
+import jdk.internal.org.objectweb.asm.Opcodes.ASM4
+import org.jetbrains.kotlin.metadata.jvm.deserialization.JvmNameResolver
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNull
+import org.junit.Test
+import java.io.File
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.zip.ZipFile
+import kotlin.text.Charsets.UTF_8
+
+class JarContentTest {
```

Comment:
Could you please move this test to `libraries/kotlinx-metadata/jvm/test`? This way you'll be testing the actual `kotlinx-metadata-jvm` artifact.

Reply:
Since `ClassMapperLite` is under `core/metadata.jvm/` I somehow thought that was the right place. Migrated.